### PR TITLE
Add contact contact form

### PR DESCRIPTION
**Pull request by Ridalord**
In this section of my portfolio, I added the following features:
- The contact form that send allows a user to send a message to the potfolio owner
- used flex properties on the contact form section
- Implemented form validations

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -228,6 +228,22 @@ <h2 class="about-header">About Myself</h2>
...
+          <div class="contact-intro d-flex">
+            <h2 class="contact-header">Contact me</h2>
+            <p class="contact-text">if you have an application you are interested in developing, a feature that you need built or
+              a project that needs coding. I'd love to help with it!</p>
+          </div>
+          <form class="contact-form d-flex">
+            <input type="text" name="name" value="Lorem Ipsum" placeholder="Name" required >
+            <input type="email" name="email" value="loremipsum@gmail.com" placeholder="email" required>
+            <textarea class="contact-message" name="message" placeholder="Write your message here.." required></textarea>
+            <!-- <input class="contact-message" type="text" name="message" placeholder="Write your message here..." required> -->
```
Comment (zieeco) [inline index.html]: - Hey, let's remove this commented code to make our codebase clean and maintainable.

File: index.html [hunk 2]
```diff
@@ -228,6 +228,22 @@ <h2 class="about-header">About Myself</h2>
...
       </section>
+      <section class="contact">
+        <div class="contact-wrap d-flex">
+          <div class="contact-intro d-flex">
+            <h2 class="contact-header">Contact me</h2>
+            <p class="contact-text">if you have an application you are interested in developing, a feature that you need built or
+              a project that needs coding. I'd love to help with it!</p>
+          </div>
+          <form class="contact-form d-flex">
+            <input type="text" name="name" value="Lorem Ipsum" placeholder="Name" required >
```
Comment (zieeco) [inline index.html]: - Hey, let's add the missing max length attribute to this input to complete the contact form. see the project requirement for the details.

File: index.html [hunk 3]
```diff
@@ -228,6 +228,22 @@ <h2 class="about-header">About Myself</h2>
...
+            <input type="email" name="email" value="loremipsum@gmail.com" placeholder="email" required>
+            <textarea class="contact-message" name="message" placeholder="Write your message here.." required></textarea>
```
Comment (zieeco) [inline index.html]: - Hey, let's add the missing max length attribute to this textarea to complete the contact form. see the project requirement for the details.

File: index.html [hunk 4]
```diff
@@ -228,6 +228,22 @@ <h2 class="about-header">About Myself</h2>
...
         </ul>
       </section>
+      <section class="contact">
+        <div class="contact-wrap d-flex">
+          <div class="contact-intro d-flex">
+            <h2 class="contact-header">Contact me</h2>
+            <p class="contact-text">if you have an application you are interested in developing, a feature that you need built or
+              a project that needs coding. I'd love to help with it!</p>
+          </div>
+          <form class="contact-form d-flex">
```
Comment (zieeco) [inline index.html]: - Hey, let's ensure our form uses the action attribute and the post method to integrate with [formspree service](https://formspree.io/html) to make the form submittable. kindly fix this.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,






Great job so far! 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — COMMENTED** [review]
Hi @Ridalord @manzitresor @Kgomotso196 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You submitted a merged pull request. Please do not merge a pull request until your work has been approved. Kindly submit a pull request that references the changes suggested by the first reviewer. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**